stock_indicators/indicators/common/helpers.py

```
@@ -9,30 +9,55 @@
+            # Try to find the specific overloaded method first
+            try:
+                condense_method = CsIndicator.Condense.Overloads[CsIEnumerable[cs_results_type]]
+            except TypeError:
+                # Fall back to generic utility method
+                condense_method = CsResultUtility.Condense[cs_results_type]
+
+            condensed_results = condense_method(CsList(cs_results_type, self._csdata))
+            return self.__class__(condensed_results, self._wrapper_class)
+
+        except Exception as e:
+            raise RuntimeError(f"Failed to condense results: {e}") from e
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)